lib/defaults.js

```diff
@@ -104,7 +104,7 @@ var defaults = {
 
 defaults.headers = {
   common: {
-    'Accept': 'application/json, text/plain, */*'
+    'Accept': '*/*'
```

Comment:
Any reason we need this change? Any issue you can tag for this change would be good.

Reply:
I mean */* covers all types right, so the other types seem redundant.